### PR TITLE
Add templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by ryanjbaxter**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: waiting-for-triage
+assignees: ''
+
+---
+
+**Describe the bug**
+Please provide details of the problem, including the version of Spring Cloud that you
```
Comment (meltsufin) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please provide details of the problem, including the version of Spring Cloud GCP that you
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: waiting-for-triage
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (meltsufin) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Can we also ask them to tell us which version they're currently on here?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ryanjbaxter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The version of Spring Cloud would correlate with the version of GCP.  Some people might not know how to figure out the exact project version

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (ryanjbaxter) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Done

**Pull request merged**